nltk/test/unit/test_seekable_unicode_stream_reader.py

```diff
@@ -96,41 +103,11 @@ def test_reader():
                 pass
 
 
-# nose shows the whole string arguments in a verbose mode; this is annoying,
```

Comment:
this seems to no longer be true with pytest, so i added this back with the other strings